tests/sentry/backup/test_releases.py

```diff
@@ -129,9 +129,9 @@ def test_at_24_4_0(self):
             with open(tmp_path, "rb") as f:
                 import_in_global_scope(f, printer=NOOP_PRINTER)
 
-    def test_at_24_3_0(self):
```

Comment:
should we delete the test_at_24_3_0.pysnap?

Reply:
It is - it's just that the diff here is odd and offset by 1. Look at the new file - only 24_4_N, and 24_5_0.

Reply:
ah misread that, thanks for clarifying